src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    public static final int AMOUNT_OF_COLORS = Color.values().length;
```

Comment:
I do not see a reason in having it as a constant, constants are usually some custom things and values that are not clear why they are where they are if they would not be named. Let's take a closer look at `100` that you had, it is not very readable if it's just plain `100` in the code, but if you name it as a `max number` or something like that - it becomes obvious.

so let's just put it back as a method variable